### ISSUE TITLE
How can I get originalObject in selected event

### ISSUE THREAD

**Issue by saostad** [oferh/ng2-completer#234]
Hi.
How can I get originalObject in selected event?

for example in your example:

`
protected searchData = [
    { color: 'red', value: '#f00' },
    { color: 'green', value: '#0f0' },
    { color: 'blue', value: '#00f' },
    { color: 'cyan', value: '#0ff' },
    { color: 'magenta', value: '#f0f' },
    { color: 'yellow', value: '#ff0' },
    { color: 'black', value: '#000' }
  ];

  constructor(private completerService: CompleterService) {
    this.dataService = completerService.local(this.searchData, 'color', 'color');
  }
`
`
<ng2-completer [(ngModel)]="searchStr" [datasource]="dataService" [minSearchLength]="0"></ng2-completer>
`

Thanks.

**Comment by rajeshtandukar** [comment]
Same Question here!

**Comment by tapahbl4** [comment]
#233 But not working

**Comment by saostad** [comment]
I use this and works for me:

(selected)="onItemSelect($event)"

and

onItemSelect(selected:CompleterItem){
    if(selected)
      this.selectedItem = selected.originalObject._id;
  }

**Comment by beyondJohn** [comment]
I agree with saostad's solution to the issue. But you must import the CompleterItem interface. 

`import { CompleterService, CompleterData, CompleterItem } from 'ng2-completer';`
